scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,72 @@
+exitstatus=$?
+if [[ $exitstatus -ne 0 ]]; then
+    echo "waiting finschia node is timeout"
+    exit 1
+fi
+# send transaction to node
+cur_path=`pwd`
+
+artifacts_path="artifacts"
+
+# This is a function that checks if executeMsg is successful.
+check_run_info() {
+  local res="$1"
+  local msg="$2"
+	if [[ "$res" == *"failed"* ]]; then
+		echo -e "$msg$res"
+		exit 1
+	fi
+}
+
+#*** store collection contract ***
+result=$(fnsad tx wasm store ${artifacts_path}/collection.wasm --node ${URL} --from ${FROM_ACCOUNT} --gas 10000000 ${CHAIN_OPTION})
+
+## confirm a result of `store`
+raw_log=$(echo ${result} | jq .raw_log)
+check_run_info "${raw_log}" "store: "
+
+#*** instantiate collection contract ***
+result=$(fnsad tx wasm instantiate 1 '{"name":"collection_name","uri":"collection_uri","meta":"collection_meta", "owner":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8"}' --label collection1 --admin link146asaycmtydq45kxc8evntqfgepagygelel00h  --from link146asaycmtydq45kxc8evntqfgepagygelel00h --node ${URL} ${CHAIN_OPTION})
+
+## confirm a result of `instantiate`
+raw_log=$(echo ${result} | jq .raw_log)
+check_run_info "${raw_log}" "instantiate: "
+
+#*** issue_nft collection contract ***
+result=$(fnsad tx wasm execute ${CONTRACT_ADDRESS} '{"issue_nft":{"name":"nft1_name","meta":"nft1_meta","owner":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8"}}' --from ${FROM_ACCOUNT} --node ${URL} --gas 10000000 ${CHAIN_OPTION})
+
+## confirm a result of `issue_nft`
+raw_log=$(echo ${result} | jq .raw_log)
+check_run_info "${raw_log}" "execute; issue_nft: "
+
+#*** mint_nft collection contract ***
+result=$(fnsad tx wasm execute ${CONTRACT_ADDRESS} '{"mint_nft":{"from":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8","to":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8","params":[{"token_type":"10000001","name":"nft1_name1","meta":"nft1_meta1"},{"token_type":"10000001","name":"nft1_name2","meta":"nft1_meta2"}]}}' --from ${FROM_ACCOUNT} --node ${URL} --gas 10000000 ${CHAIN_OPTION})
+
+## confirm a result of `mint_nft`
+raw_log=$(echo ${result} | jq .raw_log)
+check_run_info "${raw_log}" "execute; mint_nft: "
+
+#*** send_nft collection contract ***
+result=$(fnsad tx wasm execute ${CONTRACT_ADDRESS} '{"send_nft":{"from":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8","to":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8","token_ids":["1000000100000001"]}}' --from ${FROM_ACCOUNT} --node ${URL} --gas 10000000 ${CHAIN_OPTION})
+
+## confirm a result of `send_nft`
+raw_log=$(echo ${result} | jq .raw_log)
+check_run_info "${raw_log}" "execute; send_nft"
+
+#*** burn_nft collection contract ***
+result=$(fnsad tx wasm execute ${CONTRACT_ADDRESS} '{"burn_nft":{"from":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8","token_ids":["1000000100000001"]}}' --from ${FROM_ACCOUNT} --node ${URL} --gas 10000000 ${CHAIN_OPTION})
```

Comment:
ditto

Reply:
fixed here
[fix: change address to variable specification instead of hard code in…](https://github.com/Finschia/finschia-wasm/pull/3/commits/409bc9e4f57487615926518589c1c78926607913)